### ISSUE TITLE
No more updated ??

### ISSUE THREAD

**Issue by DrSaad77** [Sycnex/Windows10Debloater#302]
It is really sad this amazing creation is no more updated!! It is the fisrt thing I run on my system right after Windows installation and drivers install. Really sad... There is Farag2 version, but got no GUI....

**Comment by farag2** [comment]
@DrSaad77 , wait a little bit more. Everything will be. It's too difficult to code together without a designer and so on. No ETA but may be very first beta in late 2020.
And the 5.0 script version will be relased soon. :)

**Comment by DrSaad77** [comment]
Thx you very much for your work, and I dont doubt it is hard to code this whole project... with or even without GUI...
I am very glad to see you around here, and to know that you are active on the project 😉
Hope to can use your new version soon, until then I will stick with Sycnex old one for the time being...
PS: I know Sycnex one dont debloat everything on 2004, but it ll do 😊

**Comment by Sycnex** [comment]
I just don’t have a lot of time to be able to keep updating the project. I have something in development that I will end up releasing sometime this year. However, it just depends on how much free time I have. 

If you check the test branch, you can see the new GUI I have uploaded about 2 or 3 weeks ago. Feel free to play with it.

**Comment by fredenocs** [comment]
I am happy to hear this news.  

This shouldn't be used or ran on home systems, without knowledge of impact.  Also with each windows 10 feature update, MS changes things, taken on more development.  This doesn't save lives or improve the overall health of a system.  

I relied on your project for 1803, 1809, 1903, since then I did my own.  I run a PowerShell script I created for imaging a machine.  If the PC is granted ENTERPRISE licensing, this step is not really needed.  Basic home users STOP using this NOW.

**Comment by farag2** [comment]
@Sycnex, recently checked your dev version. So you cannot unpin Start tiles like before using

```powershell
    % { $_.Verbs() } |
    ? { $_.Name -match 'Un.*pin from Start' } |
    % { $_.DoIt() }
```
this method is obsolete and cannot be invoked. The only way to do it (without using syspin.exe) is to use

https://github.com/farag2/Windows-10-Setup-Script/blob/6ef64fa66cffaf3bf88c022b3bc669fcad78bd9b/Win%2010%202004.ps1#L2072

**Comment by DrSaad77** [comment]
> @Sycnex, recently checked your dev version. So you cannot unpin Start tiles like before using
> 
> ```powershell
>     % { $_.Verbs() } |
>     ? { $_.Name -match 'Un.*pin from Start' } |
>     % { $_.DoIt() }
> ```
> 
> this method is obsolete and cannot be invoked. The only way to do it (without using syspin.exe) is to use
> 
> https://github.com/farag2/Windows-10-Setup-Script/blob/6ef64fa66cffaf3bf88c022

**Comment by DrSaad77** [comment]
> @Sycnex, recently checked your dev version. So you cannot unpin Start tiles like before using
> 
> ```powershell
>     % { $_.Verbs() } |
>     ? { $_.Name -match 'Un.*pin from Start' } |
>     % { $_.DoIt() }
> ```
> 
> this method is obsolete and cannot be invoked. The only way to do it (without using syspin.exe) is to use
> 
> https://github.com/farag2/Windows-10-Setup-Script/blob/6ef64fa66cffaf3bf88c022b3bc669fcad78bd9b/Win%2010%202004.ps1#L2072

I am using Sycnex script on 2004 to debloat crappy stuff like Candy or OneDrive, which is super heavy I think...
About unpin Tiles dont use it much, I reset Start Menu, with Backup Start Menu Layout (Sordum soft) to get Minimum Tiles and add the ones I use...
Btw, I tried ur script method, and then had no more Start Menu available at all! 

My Bad: I had no more Start Menu Because I did unpin Tiles, and had to add at least 1 thing to it, for Start Menu to show up again...
Btw, I took a lot a the script, and find out a lot a thing, I would like to keep, but they are all removed/kicked out of the system! at the moment , had not time to investigate them 1 by 1. There are a lot of powerfull and nice stuff into the script, but definitely it should be investigate 1 by 1 to know what to debload or no (as recommanded on your page), and it also custom a bit the look of Windows I found out, like Removing "Share" from Right Cick Context Menu...
Did you take a look to the work "FreeTimeTech" did to ease the run of your debloating script?? interesting initiative...
A lot are into the script, Debloating + Custom, hope one day for it to get an GUI, to ease is use for everybody...
Yeah yeah, I know an GUI is a lot of work, but life is hope 😉
